### PR TITLE
Remove .ForEach Extension Method and Release 0.3.0

### PR DESCRIPTION
**Pull request by tautropfli**
[`.ForEach`](https://github.com/messerli-informatik-ag/funcky/blob/d74d44b399fccfaa6ef25879be0d55b5c87505ef/Funcky/Extensions/EnumerableExtensions.cs#L41-L47) from [Funcky](https://github.com/messerli-informatik-ag/funcky) should be used instead

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -18,3 +18,6 @@
 
 ## 0.2.3
 - Add `ForEachAsync` extension for `IEnumerable`
+
+## 0.3.0
+- The extension method `ForEach` for `IEnumerable` has been removed. Use the method of the same name from [Funcky](https://github.com/messerli-informatik-ag/funcky/blob/master/Funcky/Extensions/EnumerableExtensions.cs) instead (The method is called `Each` in versions before 2.0).
```
Comment (janhohenheim) [inline changelog.md]: "is called" -> "was called", "2.0" -> "2.0.0"

File: changelog.md [hunk 2]
```diff
@@ -18,3 +18,6 @@
 
 ## 0.2.3
 - Add `ForEachAsync` extension for `IEnumerable`
+
+## 0.3.0
```
Comment (janhohenheim) [inline changelog.md:22]: You've specified a new release without changing the version in the `.csproj`

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (tautropfli) [reply changelog.md]: acf4ce8577e47557fd48bf31688134c2fc154595

File: changelog.md (hunk 2 above)
Comment (tautropfli) [reply changelog.md:22]: dc15d640aff0a8f96fd9b452080620bce44e349e and 687adc0

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**